### PR TITLE
feat: design changes for ios release

### PR DESCRIPTION
**Pull request by jmars**
Some design changes that were requested by the editorial team ahead of the iOS release.
See:
https://nidigitalsolutions.jira.com/browse/REPLAT-8119
https://nidigitalsolutions.jira.com/browse/REPLAT-10181
https://nidigitalsolutions.jira.com/browse/REPLAT-10562
https://nidigitalsolutions.jira.com/browse/REPLAT-10633
https://nidigitalsolutions.jira.com/browse/REPLAT-10636
https://nidigitalsolutions.jira.com/browse/REPLAT-10568
https://nidigitalsolutions.jira.com/browse/REPLAT-10566

I have tried to avoid any changes touching web articles in order not to effect the ongoing work there, however slices has no web overrides so I have not added any there.

### CODE REVIEW

File: packages/edition-slices/src/tiles/tile-e/index.js [hunk 1]
```diff
@@ -30,9 +40,11 @@ const TileE = ({ onPress, tile, breakpoint = editionBreakpoints.small }) => {
         style={styles.imageContainer}
         uri={crop.url}
         fill
+        hasVideo={hasVideo}
       />
       <TileSummary
         headlineStyle={styles.headline}
+        summary={getTileSummary(tile, withMoreTeaser ? 800 : 125)}
```
Comment (infreezer) [inline packages/edition-slices/src/tiles/tile-e/index.js]:
```suggestion
        summary={getTileSummary(tile, 125)}
```
We aren't using TileE with `withMoreTeaser`, so we don't need that tile to support it.

File: packages/edition-slices/src/tiles/tile-p/index.js [hunk 2]
```diff
@@ -31,9 +32,8 @@ const TileP = ({ onPress, tile }) => {
         rounded
         resizeMode="cover"
       />
+      <Byline ast={tile.article.bylines} bylineStyle={styles.bylineOpinion} />
```
Comment (infreezer) [inline packages/edition-slices/src/tiles/tile-p/index.js]: According to the [story](https://nidigitalsolutions.jira.com/browse/REPLAT-10633): `Journalist Name must be in All Caps`
image

File: packages/edition-slices/src/tiles/tile-e/index.js (hunk 1 above)
Comment (jmars) [reply packages/edition-slices/src/tiles/tile-e/index.js]: Is TileE not the one mentioned in the ticket? https://nidigitalsolutions.jira.com/browse/REPLAT-10562 , or should it only have a summary on mobile?

Comment (infreezer) [reply packages/edition-slices/src/tiles/tile-e/index.js]: It is the one. We just don't need withMoreTeaser functionality. 
For tablet we are using TileE for [support1](https://github.com/newsuk/times-components/blob/master/packages/provider-queries/src/section_fragment.graphql#L458).
Support1 doesn't have summary in the query. So there is no summary for TileE for tablet.
image

Comment (jmars) [reply packages/edition-slices/src/tiles/tile-e/index.js]: Sorry I misunderstood what you meant here but that makes sense, it doesn't ever have extended teaser content so there is no need for the prop or check. Updated the PR. :)

**Pull request merged**